src/main/kotlin/org/rust/cargo/runconfig/Utils.kt

```diff
@@ -24,7 +24,7 @@ fun CargoCommandLine.mergeWithDefault(default: CargoCommandConfiguration): Cargo
         this
 
 fun RunManager.createCargoCommandRunConfiguration(cargoCommandLine: CargoCommandLine): RunnerAndConfigurationSettings {
-    val runnerAndConfigurationSettings = createRunConfiguration(cargoCommandLine.command, CargoCommandConfigurationType().factory)
+    val runnerAndConfigurationSettings = createRunConfiguration(cargoCommandLine.name, CargoCommandConfigurationType().factory)
```

Comment:
Maybe add `name` parameter into `createCargoCommandRunConfiguration` to avoid mutable state in `CargoCommandLine`?
We can make it nullable with `null` as default value and pass name only when it should differ from command name. In other cases use `cargoCommandLine.command`